bchrpc/server.go

```diff
@@ -1020,6 +1094,12 @@ func (s *GrpcServer) GetUnspentOutput(ctx context.Context, req *pb.GetUnspentOut
 		blockHeight = mining.UnminedHeight
 		scriptPubkey = tx.MsgTx().TxOut[req.Index].PkScript
 		coinbase = blockchain.IsCoinBase(tx)
+
+		// check if this txn is possibly an SLP transaction
+		_, err = v1parser.ParseSLP(tx.MsgTx().TxOut[0].PkScript)
```

Comment:
Is there any chance of getting here with a tx with no outputs? Seems we should have a length check on the outputs.

Reply:
done.